.github/workflows/README.md

```diff
@@ -83,16 +83,34 @@ To test new workflows, we recommend the following pattern:
 
 1) Fork the Beam repo
 2) Add your proposed workflow to the main branch of your fork.
-3) Run the workflow in the [Actions tab](https://github.com/apache/beam/actions) of your fork using the `Run workflow` button
+3) Run the workflow in the [Actions tab](https://github.com/apache/beam/actions) of your fork using the `Run workflow` button.
+
+Note: most workflows use [self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners)
+with the main and ubuntu labels to execute ([example](https://github.com/apache/beam/blob/5a54ee6ddd8cb8444c41802929a364fe2561001e/.github/workflows/beam_PostCommit_Go_Dataflow_ARM.yml#L41)).
+If you are testing on a fork, you likely will not have self-hosted runners set up.
+To work around this, you can start using hosted runners and then switch over when you're ready to create a PR.
```

Comment:
... for example, changing `runs-on: [self-hosted, ubuntu-20.04, main]` to `runs-on:  ubuntu-20.04`

may want to be specific if people not familiar with github actions

Reply:
good call - done